### PR TITLE
Fixed #1638 (Update Hyperparameters for 0.3.0)

### LINKED ISSUE #1638: Update Hyperparameters for 0.3.0

### PR DESCRIPTION
**Pull request by kdavis-mozilla**

### CODE REVIEW

**Comment by lissyx** [comment]
@kdavis-mozilla We might want to remove the hacks `FIXME` in `tc-tests-utils.sh`

**Comment by kdavis-mozilla** [comment]
@lissyx I'll remove them and hope for the best!

**Comment by reuben** [comment]
Either increasing the number of epochs or adjusting the test text should be enough to get green tests.

**Comment by lissyx** [comment]
> Either increasing the number of epochs or adjusting the test text should be enough to get green tests.

Looking at the failures, I think it's a bit of both: changing training on LDC93S1 sample to get better overfiting, and updating production expected text.

BTW we might want to also do a few intermittence verifications once we have a green PR, with lots of tests retriggers to ensure stability.

**Comment by kdavis-mozilla** [comment]
Seems like there are other ifra problems too, e.g. the Docker build always fails, that are not related to this PR.

**Comment by lissyx** [comment]
> Seems like there are other ifra problems too, e.g. the Docker build always fails, that are not related to this PR.

Yeah, don't worry about this, my code assumed that people would send PRs from their own repos, I'd need to fix it but it's not a big big issue :)

File: tc-tests-utils.sh [hunk 1]
```diff
@@ -135,25 +135,28 @@ assert_shows_something()
 
 assert_correct_ldc93s1()
 {
-  # FIXME: RE-ENABLE AFTER FIXING LM HYPERPARAMETERS
-  # assert_correct_inference "$1" "she had your dark suit in greasy wash water all year"
+  assert_correct_inference "$1" "she had your dark suit in greasy wash water all year"
+  assert_working_inference "$1" "she had"
```
Comment (lissyx) [inline tc-tests-utils.sh]: Don't forget to remove those `assert_working_inference`

**Pull request merged**